src/components/App.js

```diff
@@ -2,12 +2,18 @@
 import './App.css';
 import Big from 'big.js';
 import Header from './Header';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
 
 function App() {
   const x = new Big(123.4567);
   return (
     <div className="App">
       <Header title="Math Magicians" />
+      <div className="calculator">
+        <Display />
+        <ButtonPanel />
+      </div>
     </div>
```

Comment:
- [ ] [REQUIRED] The App component should have its children components wrapped in [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax) short syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/)). This will allow you to group a list of children elements without adding extra nodes to the DOM.